Modules/Devices/src/Devices.Infrastructure/Persistence/Database/DevicesDbContext.cs

```diff
@@ -173,5 +173,7 @@ protected override void OnModelCreating(ModelBuilder builder)
         builder.HasDefaultSchema("Devices");
 
         builder.ApplyConfigurationsFromAssembly(typeof(DeviceEntityTypeConfiguration).Assembly);
+
+        builder.Entity<IdentityDeletionProcessAuditLogEntry>().Property(e => e.MessageKey).HasConversion<string>();
```

Comment:
This should be moved to the `IdentityDeletionProcessEntityTypeConfiguration`. You just have to modify it a little bit in order to be able to do so:

```suggestion
        builder.Property(e => e.MessageKey).HasConversion<string>();
```

Reply:
Changes made.